dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/general/SfdrGeneral.kt

```diff
@@ -0,0 +1,14 @@
+package org.dataland.datalandbackend.model.sfdr.categories.general
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.dataland.datalandbackend.model.sfdr.categories.general.subcategories.SfdrGeneralGeneral
+
+/**
+ * --- API model ---
+ * Fields of the category "General" of the sfdr framework.
+ */
+
+data class SfdrGeneral(
+    @JsonProperty(value = "general", required = true)
```

Comment:
Why is "value = general" needed in the annotation?

Reply:
Because we ran into an error in the devtools where it did not compile due to an jackson error. To mitigate the erorr we had to specify a value and it seems that this made it's way into the autogenerated code. However it is no longer necessary and should be gone now.

Reply:
Resolving this because of this:

https://stackoverflow.com/questions/53191468/no-creators-like-default-construct-exist-cannot-deserialize-from-object-valu